packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js

```diff
@@ -71,6 +72,13 @@ export const getData = (state, ownProps) => {
   const excluded = filter(x => !exclude.includes(x.label))
   const toDropdown = map(x => ({ label: buildDisplay(x), value: x }))
   const toGroup = curry((label, options) => [{ label, options }])
+  const toPit = x => [{ label: `BCH PIT Address (${x})`, value: x }]
+
+  const pitAddress = selectors.components.send.getPaymentsAccountPit(
+    'BCH',
+    state
+  )
+  const hasPitAddress = Remote.Success.is(pitAddress)
```

Comment:
👍

Reply:
Assuming this safely allows the send dropdown to still render even if API call fails